src/chrome/content/rules/Mifile.cn.xml

```diff
@@ -0,0 +1,28 @@
+<!--
+	Invalid Certificate (360buyimg.com)
+		img04, img05
+-->
+<ruleset name="Mifile.cn (partial)">
+	<target host="s01.mifile.cn" />
+		<test url="http://s01.mifile.cn/js/base.min.js?v2016d30" />
+	<target host="c1.mifile.cn" />
+		<test url="http://c1.mifile.cn/f/i/15/stat/js/xmst.js" />
+
+	<target host="i1.mifile.cn" />
+		<test url="http://i1.mifile.cn/a1/T1_WhgBb_T1RXrhCrK.jpg?width=60&#x26;height=60" />
+	<target host="i2.mifile.cn" />
+		<test url="http://i2.mifile.cn/a1/T1_WhgBb_T1RXrhCrK.jpg?width=60&#x26;height=60" />
+	<target host="i3.mifile.cn" />
+		<test url="http://i3.mifile.cn/a1/T1_WhgBb_T1RXrhCrK.jpg?width=60&#x26;height=60" />
```

Comment:
Please remove unnecessary parameters like `?v2016d30` and `?width=60&#x26;height=60`.